cluster/gpu-player.yaml

```diff
@@ -21,7 +21,7 @@ spec:
             topologyKey: kubernetes.io/hostname
       containers:
       - name: player
-        image: gcr.io/$PROJECT/minigo-gpu-player:0.10
+        image: gcr.io/$PROJECT/$GPU_PLAYER_CONTAINER:$VERSION_TAG
```

Comment:
This seems like a good way to inadvertently push a new version when you only want to change e.g. the parallelism.

Reply:
I think this is MUCH better and safer than the original approach. Now, all you need to do is say `export VERSION_TAG=0.10` and then that tag is used everywhere consistently -- whether via `make` or deploying a cluster.

More likely, I can see people forgetting to edit the YAML and then pushing a downgrade for their cluster.

Reply:
Ok, i definitely agree this is better :)
But there's still the problem that it's defined with a default in the Makefile *and* in common.sh.  If i don't run common.sh to export things, then i'm gonna end up with the wrong default.